server/services/v1/search_reader.go

```diff
@@ -18,16 +18,17 @@ import (
 	"context"
 
 	jsoniter "github.com/json-iterator/go"
+	tsApi "github.com/typesense/typesense-go/typesense/api"
+
```

Comment:
remove the line and run `go imports`.

Reply:
apparently this is after goimports, it is grouping imports by package. Is there a particular configuration that we follow for goimports?

Reply:
So we remove the extra lines and then run `go imports`. This way there are only two groups of imports.